czishrink/netczicompress.Installer/CziShrinkMSI/Package.wxs

```diff
@@ -0,0 +1,49 @@
+﻿<!--
+SPDX-FileCopyrightText: 2023 Carl Zeiss Microscopy GmbH
+SPDX-License-Identifier: GPL-3.0-or-later
+-->
+<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:ui="http://wixtoolset.org/schemas/v4/wxs/ui">
+  <?include CziShrinkVariables.wxi ?>
+  <Package Name="$(var.FullName)"
+           Manufacturer="$(var.ManufacturerFullName)"
+           Version="!(bind.fileVersion.CziShrinkExe)"
+           UpgradeCode="6d400b13-c2d8-4490-87d5-e5492c7f8277">
+    <MajorUpgrade DowngradeErrorMessage="!(loc.DowngradeError)" />
+
+    <!-- We want to highly compress the cab file and embed it into the executable-->
+    <MediaTemplate CompressionLevel="high" EmbedCab="yes" />
+
+    <ui:WixUI
+      Id="WixUI_InstallDir"
+      InstallDirectory="INSTALLFOLDER"/>
+
+    <WixVariable Id="WixUILicenseRtf" Value="license.rtf" />
+    <!-- Unfortunately it appears that windows installer needs explicit CodePages to be set and has issues w/ Utf-8.
```

Comment:
This comment can be removed because it is no longer relevant. We do not have an artifacts rtf file which is displayed....